Cabal/src/Distribution/Simple/Setup/Haddock.hs

```diff
@@ -505,6 +514,11 @@ haddockProjectOptions _showOrParseArgs =
      "location of Haddocks static / auxiliary files"
      haddockProjectLib (\v flags -> flags { haddockProjectLib = v})
      (reqArgFlag "DIR")
+
+    ,option "" ["odir"]
+     "Generate haddock documentation into this directory. (Tech preview, subject to change.)"
```

Comment:
```suggestion
     "Generate haddock documentation into this directory. This flag is provided as a technology preview and is subject to change in the next releases."
```

Reply:
oh, are you sure that something as simple as `--haddock-odir` should be a _tech preview_.  It is consistent with other flags, so I wouldn't bother with it.  Unlike the whole `hadock-project` which deserve to be in that category.

Reply:
Because you never know in advance what may happen, and it costs us nothing.